src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOpBGPAdd.java

```diff
@@ -45,4 +46,14 @@ public void visit( final LogicalPlanVisitor visitor ) {
 		visitor.visit(this);
 	}
 
+	@Override
+	public String toString(){
+
+		return "> bgpAdd ( "
+				+ bgp.toString()
+				+ ", "
+				+ LogicalPlanUtils.printStringOfFm( fm )
```

Comment:
As mentioned above, also for the `FederationMember` objects we extend each class that implements `FederationMember` with an overridden `toString()` method and, then, rely on that method here.